clang/runtime/dpct-rt/include/dpct/shmem_utils.hpp

```diff
@@ -0,0 +1,45 @@
+//==--------------- shmem_utils.hpp ---------------*- C++ -*----------------==//
+//
+// Copyright (C) Intel Corporation
+// SPDX-License-Identifier: Apache-2.0 WITH LLVM-exception
+// See https://llvm.org/LICENSE.txt for license information.
+//
+//===----------------------------------------------------------------------===//
+
+#ifndef __DPCT_SHMEM_UTILS_HPP__
+#define __DPCT_SHMEM_UTILS_HPP__
+
+namespace dpct::shmem {
+
+/// Setup Operations
```

Comment:
```suggestion
/// Setup Operations
```
what end user can get from this ?

Reply:
Added these comments to serve as logical groupings of APIs for helper functions as there are many nvSHMEM APIs